### DISCUSSION TITLE
What is the key to open the map?

### DISCUSSION THREAD

**Discussion by youngjae142** [praydog/REFramework#335] [General Questions] [answered]
I couldn't find out the key to open the map. (Oculus Quest 2)
Plz help!

**Comment by Gonzaxpain** [comment, marked as answer]
I couldn't find it either. On Vive apparently you can open it by moving your left hand behind your left shoulder and pressing the System button but that didn't work for me on Quest 2 via VD.

What I did was to rebind the right A button to click-minimap (touch for normal A button functionality)
Once inside the map you can move between floors by keeping the left grip and left trigger pressed and then move the left joystick up and down

**Reply by youngjae142** [reply]
@Gonzaxpain
Thx you so much!
So, how can I rebind the A button to click-minimap?
Is it in steamvr setting?

**Reply by Gonzaxpain** [reply]
Yes, you can rebind anything to your liking if you go  to Steam VR settings, for any headset.
Here is a video tutorial for Quest, it's very easy: https://www.youtube.com/watch?v=ZxQPSGNQIX8

There aren't any free buttons available for RE7 with touch controllers so for me mapping it to the Right A Button worked very well. A simple touch will use A as normal and clicking it will open the map. You'll get used to it very quickly but you can always use another button if you like, just be careful you don't disable another necessary action. In case you need to revert to the previous controls you can always choose 'default' inside SteamVR control settings and you'll be fine.

**Comment by youngjae142** [comment]
How to change the binding for Minimap (OpenXR gameplay)
> 1. Open the file `_GAMEFOLDER_\reframework\autorun\re8_vr.lua`
> 2. Revise `local action_minimap = vrmod:get_action_minimap()` line to `local action_minimap = vrmod:get_action_a_button()`

Now you can access the minimap with X button of oculus touch controller.

**Reply by praydog** [reply]
> How to change the binding for Minimap (OpenXR gameplay)
> 
> > 1. Open the file `_GAMEFORDER_\reframework\autorun\re8_vr.lua`
> > 2. Revise `local action_minimap = vrmod:get_action_minimap()` line to `local action_minimap = vrmod:get_action_a_button()`
> 
> Now you can access the minimap with X button of oculus touch controller.

This is not necessary.

1. You can open the map by moving your **_left hand_** behind your head/over your shoulder, and pressing the **_inventory button_**. The reason it's like this and not just a button press is because Touch doesn't have enough buttons, and OpenXR doesn't support Chords or timed inputs out of the box like SteamVR does.
2. You can still bind a new button to the minimap using OpenXR, see the below video.

https://user-images.githubusercontent.com/2909949/170300748-a87e0c01-8042-435a-8818-200edf68da34.mp4

**Reply by youngjae142** [reply]
@praydog 
It works perfectly!
And thx for letting me know the correct way to add a new binding!

**Reply by cpeng03d** [reply]
Hi thanks for the help.
However I do not see the **bindings** section in REFramework menu panels.

**Reply by praydog** [reply]
> Hi thanks for the help. However I do not see the **bindings** section in REFramework menu panels.

It is only present when using OpenXR. You need to use SteamVR's binding system if you're not using OpenXR.

**Comment by KenticoDevTrev** [comment]
For those coming here, using the default binding for pc (this is using steam vr) for quest 2 (so may be similar)

Map: put left hand by your ear and press y
Heal: put your right hand by your right ear then hold right grip and press right trigger
Block: push hand out like your shoving someone
Quick select: hold left grip then push direct on left joystick

I haven't gotten to the point of multiple ammo to try to figure that out... This is no where online, I got lucky on the heal figure out
